File: countdown.py
def countdown(num):
    if num == 0:
        return 0
    if num < 0:
        print(num)
        return countdown(num+1)
    else:
        print(num)
        return countdown(num-1)
    
def main():
    num = int(input("Enter a number: "))
    print("Blastoff: ")
    result = countdown(num)
    print(0)
    print("Countdown complete!")

if __name__ == "__main__":
    main()

File: Countdown.java
import java.util.Scanner;

class Countdown {
    public static int countDown(int num){
        if(num == 0){
            return 0;
        }
        else if(num < 0){
            System.out.println(num);
            return countDown(num+1);
        }
        else{
            System.out.println(num);
            return countDown(num-1);
        }
    }

    public static void main(String[] args){
        Scanner inp = new Scanner(System.in);
        System.out.print("Enter a number: ");
        int num = inp.nextInt();
        System.out.println("Blastoff: ");
        int result = countDown(num);
        System.out.printf("%d\n", result);
        System.out.println("Countdown complete.");
        inp.close();
    }
}